src/multi.c

```diff
@@ -206,8 +211,9 @@ void execCommand(client *c) {
     orig_cmd = c->cmd;
     addReplyArrayLen(c,c->mstate.count);
     for (j = 0; j < c->mstate.count; j++) {
-        c->argv_len = c->argc = c->mstate.commands[j].argc;
+        c->argc = c->mstate.commands[j].argc;
         c->argv = c->mstate.commands[j].argv;
+        c->argv_len = c->mstate.commands[j].argv_len;
```

Comment:
should we add `c->mstate.commands[j].argv_len = c->argv_len;` below/here ?  although i don't see problem here (we did not use it after exec), some kind of  inconsistency
```c
        /* Commands may alter argc/argv, restore mstate. */
        c->mstate.commands[j].argc = c->argc;
        c->mstate.commands[j].argv = c->argv;
        c->mstate.commands[j].cmd = c->cmd;
```

Reply:
i guess it's only there to make sure we free the right pointer.
but i agree it'll be nicer to sync that var back too.
please make a PR.